### ISSUE TITLE
IA not spawning

### ISSUE THREAD

**Issue by ghost** [GreuhZbug/greuh_liberation.Altis#122]
Hello all! I am in a dedicated server, and we are experiencing one strange problem. From time to time, IA's stop spawning in the big circular areas for some reason. We enter the cicle, and the bar gets full blue with any IA force opposing us. We tried to exit the cicle and re-enter again, but it happens the same, we waited for about 15-20 minutes for them to spawn in the middle of the cicle... and nothing. Anyone knows how to solve this problem? Thank you very much!

Gotta say I ported this mission to Chernarus, and we use ACE as well.

**Comment by Wyqer** [comment]
Hi there,

you say "from time to time", so it don't happen everytime?
Is there something that's always the same if this happens? Playtime, Playercount?
Maybe there are too many sectors active, so the server stop spawning. But you should notice a hint about that in the lower right corner.

Maybe you try the continued version (chernarus is available for it by the way) and if you still encounter issues, you report back in the Forum thread or in the repository of the continued version.

https://forums.bistudio.com/topic/202711-mpcti-coop-liberation-continued/

**Comment by ghost** [comment]
Hey! Thanks for your quick response.

Not everytime. A couple of days ago, we were playing 4 people to take a radio tower. We had to take down a patrol of 6 enemies, and when we finished the job and seized the tower, we moved on to the closest enemy point: a village. We approached the village and then no one was there. We continued to another village and it was empty again. Couldn't tell you the playtime, but i will keep testing it.

> Maybe there are too many sectors active, so the server stop spawning. But you should notice a hint about that in the lower right corner.

Were shall I see that hint? Which of the numbers in the lower right corner?

And I'm afraid I can't use that version of Chernarus, since we are playing another one based on the FFAA from Spain: http://steamcommunity.com/sharedfiles/filedetails/?id=877110767&searchtext=

Thanks again for your response! Hope we can fix this problem, it is really a pain in the ass :(

**Comment by Wyqer** [comment]
> since we are playing another one based on the FFAA from Spain

Then you should also ask the people, who made that version. Maybe they've changed something or messed a little thing up. As it is not the "basic" version it's hard to tell what's wrong or to help you, as I don't know what they've done.

**Comment by ghost** [comment]
Yeah, maybe you are right and I should contact the main creator. I will let you know if I we manage to fix it, but it is quite strange. We had a previous version of Chernarus without ACE (by the same editor), and we didnt have that problem.
